internal/pkg/fds/imported_service_store.go

```diff
@@ -15,38 +15,47 @@
 package fds
 
 import (
+	"github.com/openshift-service-mesh/federation/internal/pkg/config"
 	"sync"
 
 	"github.com/openshift-service-mesh/federation/internal/api/federation/v1alpha1"
 )
 
+type FederatedSvcByPeer map[string][]*v1alpha1.ExportedService
```

Comment:
I introduced this alias type for convenience.

Reply:
TBH I would prefer map, but does not really matter.

Reply:
I just noticed it's all over the place, so I thought it can be handy. But to be honest it might be more noise than help as it pretends to be more than just a map and does not encapsulate any extra behaviour. I will revert it before merging